openedx/core/djangoapps/user_api/legacy_urls.py

```diff
@@ -0,0 +1,41 @@
+"""
+Defines the URL routes for this app.
+"""
+
+from django.conf import settings
+from django.conf.urls import include, patterns, url
+from rest_framework import routers
+
+from . import views as user_api_views
+from .models import UserPreference
+
+
```

Comment:
Interestingly, this file just moved unchanged, so I don't know why GitHub doesn't see that.